src/components/EditRequest.vue

```diff
@@ -0,0 +1,203 @@
+        :btnDisabled="false"
+        @btnClicked="toSummary"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+/* eslint-disable no-unused-vars */
+import Button from "@/components/shared/Button.vue";
+import TextInput from "@/components/shared/TextInput.vue";
+import BigTextInput from "@/components/shared/BigTextInput.vue";
+import NumberInput from "@/components/shared/NumberInput.vue";
+import Item from "@/components/shared/Item.vue";
+import AddressInput from "@/components/AddressInput.vue";
+
+export default {
+  name: "EditRequestView",
+  components: {
+    TextInput,
+    Item,
+    Button,
+    BigTextInput,
+    NumberInput,
+    AddressInput
+  },
+  data() {
+    return {
+      errorMsg: false,
+      addressError: false,
+      phoneNumberError: false,
+      zeroItemsError: false,
+      itemNameError: false,
+      paymentSolutionError: false,
+      items: [],
+      address: {},
+      phoneNr: "",
+      arrivalDesc: "",
+      paymentSolution: "",
+      id: ""
+    };
+  },
+  methods: {
+    getRequest() {
+      return new Promise((resolve, reject) => {
+        resolve(
+          this.$store.getters.requests.find(
+            request => request.id === this.$route.params.id
+          )
+        );
+      });
+    },
+    updateAddress(event) {
+      const { value } = event.target;
+      this.addressError = false;
+      this.address = value;
+    },
+    updatePhoneNumber(event) {
+      const { value } = event.target;
+      this.phoneNumberError = false;
+      this.phoneNr = value;
+    },
+    updateArrivalDescription(value) {
+      this.arrivalDesc = value;
+    },
+    updatePaymentSolution(event) {
+      const { value } = event.target;
+      this.paymentSolutionError = false;
+      this.paymentSolution = value;
+    },
+    deleteItem(index) {
+      this.items.splice(index, 1);
+      this.errorMsg = false;
+    },
+    addItem(index) {
+      if (this.items[index].itemName.length > 0) {
+        this.items[index].added = true;
+        this.errorMsg = false;
+      } else {
+        this.itemNameError = true;
+      }
+    },
+    renderNewItem() {
+      this.items.push({
+        itemName: "",
+        count: 1,
+        added: false
+      });
+      this.errorMsg = false;
+      this.zeroItemsError = false;
+    },
+    updateItemName(input, index) {
+      this.itemNameError = false;
+      this.items[index].itemName = input;
+    },
+    incrementItemCount(index) {
+      this.items[index].count += 1;
+    },
+    decrementItemCount(index) {
+      if (this.items[index].count > 1) {
+        this.items[index].count -= 1;
+      }
+    },
+    toSummary() {
+      const itemsMapped = this.items.map(item => item.added);
+      if (this.paymentSolution.length <= 0) {
+        this.paymentSolutionError = true;
+        return;
+      }
+      if (itemsMapped.length > 0 && itemsMapped.every(Boolean)) {
+        const localAddress = this.getAddress ? this.getAddress : this.address;
```

Comment:
```suggestion
        const localAddress = this.getAddress || this.address;
```

Litt kortere, og kanskje litt lettere å lese?

Reply:
👍🏼 fixed